src/www/index.html

```diff
@@ -38,7 +140,429 @@ <h4 align="center"> Icebreaker App</h4>
+        <h3>Create a room</h3>
+    </div>
+
+
+    <div role="main" class="ui-content">
+        <p>Instructions: .</p>
+        <p>> To create a room, please enter a key in the input box and then hit the submit button </p>
+    </div><!-- /content -->
+
+
+
+    <form id="JoinForm2" class="form-horizontal" action="#user_info">
+        <div class="form-group">
+            <p>
+                <label for="cname2">Please enter your pin</label>
+                <input id="cname2" name="room_key" type="number" class="form-control required" maxlength="4">
+            </p>
+
+            <a href="#user_info" class="ui-btn ui-corner-all ui-shadow ui-btn-b ui-btn-inline">Create a room</a>
+
+        </div>
+    </form>
+
+
+    <div id="result"></div>
+
+
 </div>
 
+<!--------------------------------------------Page 3 begin here------------------------------------------------------>
+
+<div data-role="page" id="user_info" data-theme="a" align="center" class="ui-field-contain">
+
+    <div data-role="header">
+        <a href="/index.html" data-icon="delete">Cancel</a>
+        <h3>User information</h3>
+    </div>
+
+    <form id="JoinForm3" class="form-horizontal" action="#select_topics">
+        <br>
+        <br>
+        <div class="form-group">
+            <input type="text" name="text-basic" id="GetName" class="form-control required" placeholder="Enter your name..." required>
+
+
+            <input type="text" id="GetMajor" class="form-control required" placeholder="Enter your major..." required>
+
+            <br>
+            <label for="GetYear">Select your year:</label>
+            <select name="SelectElmt" id="GetYear">
+                <option value="">Select one...</option>
+                <option value="Freshman">Freshman</option>
+                <option value="Sophomore">Sophomore</option>
+                <option value="Pre-Junior">Pre-Junior</option>
+                <option value="Junior">Junior</option>
+                <option value="Senior">Senior</option>
+            </select>
+            <br>
+            <a href="#select_topics" class="ui-btn ui-corner-all ui-shadow ui-btn-b ui-btn-inline">Submit</a>
+
+            <br>
+
+        </div>
+    </form>
+
+    <div id="result"></div>
+    <footer>
+        <div data-role="footer" data-position="fixed">
+            <h4 align="center"> Icebreaker App</h4>
+        </div>
+    </footer>
+
+</div>
+
+
+<!--------------------------------------------Page 4 begin here------------------------------------------------------>
+
+<div data-role="page" id="select_topics" data-theme="a" align="center" class="ui-field-contain">
+    <div class="slideshow-container">
+        <div data-role="header">
+            <a href="/index.html" data-icon="delete">Cancel</a>
+            <h3>Select topics </h3>
+        </div>
+
+        <!-- ========================================Topic 1=================================================================-->
```

Comment:
only have a topic entry for 1 topic. Don't manually code in an entry for each topic. Instead we will an HTTP request to the get all topics endpoints.